### PR TITLE
Added anonymous telemetry helpers.

### PR DESCRIPTION
**Pull request by Alek99**

### CODE REVIEW

File: pynecone/utils.py [hunk 1]
```diff
@@ -1228,3 +1230,46 @@ def get_redis() -> Optional[Redis]:
...
+    return multiprocessing.cpu_count()
+
+
+def add_telemetry() -> Dict:
+    """Add telemetry to an event.
+
+    Returns:
+        The payload with telemetry added.
+    """
+    payload = {}
```
Comment (picklelo) [inline pynecone/utils.py:1268]: I think we can create a class `Telemetry(pc.Base)` with these fields to make it strongly typed - then we can call the `.json()` method to get this dict.

We can make a new file `telemetry.py` and define the class and this method there

File: tests/test_telemetry.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import pytest
```
Comment (picklelo) [inline tests/test_telemetry.py:1]: Thanks for adding tests

File: pynecone/telemetry.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    """Add telemetry to an event.
+
+    Returns:
+        The payload with telemetry added.
+    """
+    payload = {}
+    payload["os"] = get_os()
+    payload["cpu_count"] = get_cpu_count()
+    # Display memory in MB.
+    payload["memory"] = psutil.virtual_memory().total >> 20
```
Comment (picklelo) [inline pynecone/telemetry.py:48]: We can put this in a helper also in a follow-up

**Review by picklelo — APPROVED** [review]
Nice

File: pynecone/telemetry.py (hunk 3 above)
Comment (Alek99) [reply pynecone/telemetry.py:48]: Yeah not done yet still need to make the class aswell for telemetry

**Pull request merged**